lib/services/local/access_list.go

```diff
@@ -184,6 +184,44 @@ func (a *AccessListService) UpsertAccessList(ctx context.Context, accessList *ac
 	return upserted, nil
 }
 
+// UpdateAccessList updates an access list resource.
+func (a *AccessListService) UpdateAccessList(ctx context.Context, accessList *accesslist.AccessList) (*accesslist.AccessList, error) {
```

Comment:
The body of this function is almost the same as `UpsertAccessList`. Creating a common implementation for both will make the maintenance simpler.

Reply:
Will simplify

Reply:
done in [854509c](https://github.com/gravitational/teleport/pull/40036/commits/854509ca88f96203fdb7b0bb9ff1b56ffe1c255b)